source-dynamodb/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Build Stage
+################################################################################
+FROM golang:1.20-buster as builder
+
+WORKDIR /builder
+
+# Download & compile dependencies early. Doing this separately allows for layer
+# caching opportunities when no dependencies are updated.
+COPY go.* ./
+RUN go mod download
+
+COPY go                 ./go
+COPY source-boilerplate ./source-boilerplate
+COPY source-dynamodb    ./source-dynamodb
+
+# Run automated tests. To skip tests which interact with an external database,
+# specify `--build-arg TEST_DATABASE=no` in the Docker command line.
+ARG TEST_DATABASE=yes
+ENV TEST_DATABASE=$TEST_DATABASE
+RUN go test -v ./source-dynamodb/...
+
+# Build the connector.
+RUN go build -o ./connector -v ./source-dynamodb/...
+
+# Runtime Stage
+################################################################################
+FROM ghcr.io/estuary/base-image:v1
+
+WORKDIR /connector
+ENV PATH="/connector:$PATH"
+
+# Bring in the compiled connector artifact from the builder.
+COPY --from=builder /builder/connector ./source-dynamodb
+
+LABEL FLOW_RUNTIME_PROTOCOL=capture
+LABEL CONNECTOR_PROTOCOL=flow-capture
```

Comment:
nit: you only need `LABEL FLOW_RUNTIME_PROTOCOL=capture` now. `CONNECTOR_PROTOCOL` isn't used.